### DISCUSSION TITLE
How does this Dockle action work against multi-platform images?

### DISCUSSION THREAD

**Discussion by rlaiola** [erzz/dockle-action#22] [Q&A] [answered]
Hi, 

A few questions:

1) how does this action (and Dockle) work in the case of a multi-arch/platform image?
2) would it actually make sense to scan each specific platform of such an image or just one of them?
3) In the case of the former, how could one specify that?

Thanks a lot and congrats for the work!

**Comment by erzz** [comment, marked as answer]
Hi @rlaiola 

Sorry for the slow response - didnt even realise I had discussions enabled :)


1. As its essentially based on the contents of the file system of the image - I would say its identical in findings regardless of platform
2. Given my answer to 1, I would say just one would be fine. But then its so fast (a few seconds once the image is pulled) its just as easy to scan them all in a matrix or as part of a reusable workflow
3. I would recommend the matrix approach https://docs.github.com/en/actions/using-jobs/using-a-matrix-for-your-jobs

**Reply by rlaiola** [reply]
Thanks, @erzz! That's exactly the approach I followed. The final result can be seen here https://github.com/rlaiola/boca-docker/blob/master/.github/workflows/lint-images.yml

Cheers!

**Reply by erzz** [reply]
great stuff!
